src/endpoints/user-status/domain/models/get-user-status.input.ts

```diff
@@ -0,0 +1,28 @@
+import { GetUserStatustDto } from '../../data/dtos/userstatus.dto';
+import { Request } from '@alien-worlds/api-core';
+/**
+ * @class
+ */
+export class GetUserStatusInput {
+	/**
+	 *
+	 * @param {CandidateRequestDto} dto
```

Comment:
Just a minor thing, probably need to update the param type here to `GetUserStatustDto`.